<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderMaintance</class>
 <widget class="QWidget" name="OrderMaintance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>617</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1072</width>
    <height>617</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1072</width>
    <height>617</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Order Maintance</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_11">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0" rowspan="2" colspan="2">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>111</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_OrderNumber">
           <property name="text">
            <string>Order:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_order_number"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_CustomerName">
           <property name="text">
            <string>Customer Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_customer_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Order Type:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_OrderType"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>301</width>
          <height>90</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Label Form:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_Label_Form"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Report Form:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_Doc_Form"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4" rowspan="3">
       <widget class="QFrame" name="frame_3">
        <property name="maximumSize">
         <size>
          <width>281</width>
          <height>151</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Order Date:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDateEdit" name="dateEdit_OrderDate"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Ship Date:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDateEdit" name="dateEdit_ShipDate"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Created By:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_CreatedBy"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_Status"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2" rowspan="3">
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>90</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Carrier:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_Carrier"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_Ship_method"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Ship Method:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3" rowspan="4">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>38</width>
          <height>130</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" rowspan="3">
       <widget class="QFrame" name="frame_9">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_Store"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>Store:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_InvoiceNumber"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Invoice Num:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Vendor Num:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_Vendor_num"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1" rowspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>98</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="4" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>333</width>
          <height>48</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>334</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Adress">
      <attribute name="title">
       <string>Adress</string>
      </attribute>
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>501</width>
         <height>281</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Company Name:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Address:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>City / State / Zip Code:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="lineEdit_shp_to_City"/>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="lineEdit_shp_to_State"/>
           </item>
           <item row="3" column="4">
            <widget class="QLineEdit" name="lineEdit_shp_to_ZipCode"/>
           </item>
           <item row="4" column="2" rowspan="2">
            <widget class="QLineEdit" name="lineEdit_shp_to_Country"/>
           </item>
           <item row="6" column="2" rowspan="2">
            <widget class="QLineEdit" name="lineEdit_shp_to_ContactName"/>
           </item>
           <item row="8" column="2">
            <widget class="QLineEdit" name="lineEdit_shp_to_ContactPhone"/>
           </item>
           <item row="9" column="2">
            <widget class="QLineEdit" name="lineEdit_shp_to_TaxId"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Tax ID:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Contanct Phone:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" rowspan="2">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Contanct Name:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" rowspan="2">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Country:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="4">
            <widget class="QLineEdit" name="lineEdit_11"/>
           </item>
           <item row="0" column="1" colspan="4">
            <widget class="QLineEdit" name="lineEdit_shp_to_Company"/>
           </item>
           <item row="2" column="1" colspan="4">
            <widget class="QLineEdit" name="lineEdit_shp_to_Addres2"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Ship To Adress</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_7">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>20</y>
         <width>521</width>
         <height>281</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="0">
         <widget class="QFrame" name="frame_8">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Company Name:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Address:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>City / State / Zip Code:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="lineEdit_bill_to_City"/>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="lineEdit_bill_to_State"/>
           </item>
           <item row="3" column="4">
            <widget class="QLineEdit" name="lineEdit_bill_to_ZipCode"/>
           </item>
           <item row="4" column="2" rowspan="2">
            <widget class="QLineEdit" name="lineEdit_bill_to_Country"/>
           </item>
           <item row="6" column="2" rowspan="2">
            <widget class="QLineEdit" name="lineEdit_bill_to_ContactName"/>
           </item>
           <item row="8" column="2">
            <widget class="QLineEdit" name="lineEdit_bill_to_ContactPhone"/>
           </item>
           <item row="9" column="2">
            <widget class="QLineEdit" name="lineEdit_bill_to_TaxId"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Tax ID:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Contanct Phone:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" rowspan="2">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Contanct Name:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" rowspan="2">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Country:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="4">
            <widget class="QLineEdit" name="lineEdit_bill_to_Addres"/>
           </item>
           <item row="0" column="1" colspan="4">
            <widget class="QLineEdit" name="lineEdit_bill_to_Company"/>
           </item>
           <item row="2" column="1" colspan="4">
            <widget class="QLineEdit" name="lineEdit_bill_to_Addres2"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Bill To Adress</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Custom">
      <attribute name="title">
       <string>Custom</string>
      </attribute>
      <widget class="QFrame" name="frame_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>301</width>
         <height>191</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="3" column="0">
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>Custom 4:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Custom 1:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>Custom 3:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>Custom 5:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Custom 2:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_custom_1"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_custom_2"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_custom_3"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_custom_4"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_custom_5"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
